documentation/docs/01-routing.md

```diff
@@ -323,11 +323,10 @@ A route can have multiple dynamic parameters, for example `src/routes/[category]
 It's possible for multiple routes to match a given path. For example each of these routes would match `/foo-abc`:
 
 ```bash
+src/routes/[...catchall].svelte
```

Comment:
should you be allowed to have `[...catchall].svelte` and `[b].svelte`? It seems like `[b].svelte` will never be called, so maybe this should throw an error during build

Reply:
They would both be allowed once we implement part 2 of this with route validators, right? (And that is going to happen immediately after this PR.)

Reply:
`[...catchall]` is a rest parameter. regardless of part 2, it will match `/things/b/will/not/match`. We _could_ throw an error for `[a].js` alongside `[b].svelte`, but I think we can do that separately to avoid bloating this PR

Reply:
I assumed this was the section where we showed the priority order of routes (GitHub collapsed the context around this). It looks like `[..catchall].svelte` is still last in the priority ordering. I wonder if we should show it last here as well for consistency?